inst/tutorials/discovr_13/discovr_13.Rmd

```diff
@@ -1128,7 +1128,7 @@ There is a detailed solution to this task at [https://www.discovr.rocks/solution
 
 ```{r quiz_con_xbox, echo = FALSE}
```

Comment:
The question is poor I think because there are two valid/correct answers but the question is a bit non-specific. There are various solutions, but I think I'd favour

```
```{r quiz_con_xbox, echo = FALSE}
quiz(caption= "Setting contrasts (level 3)",
type = "learnr_checkbox",
question("What contrast codes for the variables **game** and **console** are orthogonal and valid [Select ALL valid answers]",
    answer("**game**: static (-1/2), active (1/2); **console**: Switch (-1/2), Xbox One (1/2)", correct = T, message = "It would also be valid to reverse the signs and use **game**: static (1/2), active (-1/2); **console**: Switch (1/2), Xbox One (-1/2)."),
    answer("**game**: static (0), active (1); **console**: Switch (0), Xbox One (1)", message = "You have used dummy coding, these contrasts are not orthogonal."),
    answer("**game**: static (1), active (-1); **console**: Switch (1), Xbox One (-1)", correct = T, message = "These contrasts are orthogonal and valid, but I wouldn't use them because the bs won't represent the exact differences between means (but will be proportional to them)."),
    answer("**game**: static (1), active (2); **console**: Switch (-2), Xbox One (1)", message = "These contrasts are not orthogonal."),
    correct = "Correct - well done!",
    incorrect = "Good try.",
    random_answer_order = TRUE,
    allow_retry = T
)
)
```
```

Reply:
@profandyfield Done, ready to merge.